sdks/android/Mentat/build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
     ext.build = [
             compileSdkVersion: 27,
             targetSdkVersion: 27,
-            minSdkVersion: 16
+            minSdkVersion: 19
```

Comment:
Can I get an explanation for why this is necessary?  We're not fixed _yet_, but I'd like to understand what's driving our choices.

Reply:
At the very least, this series makes `RustObject` use `AutoCloseable` which is API19+. This allows for a much prettier pattern around try-with-resources - `close` is automatically called when exiting the `try` block.